### ISSUE TITLE
Password Glitch Login Issue

### ISSUE THREAD

**Issue by Luminariah** [Healix/Gw2Launcher#437]
Each time the selected account is launched it fill in the email but then locks up the launcher (doesn't allow me to click) and it flashed that it selects the password field almost as if it's attempting to paste the password in, but it will not insert the password nor enter the game. Sits and repeats the password field selection and then locks again after a split second not even allowing me to manually enter a password. Attempted to use main page 7255 build and beta build. Cleared cache, changed how password is entered under tweaks, same issue.

**Comment by Healix** [comment]
Are you sure you're using the new version? The beta version would look the same as the previous, whereas the release version will have 7255 listed under Settings > Updates.

The only other reason it may do that is if the coordinates were previously configured are it's no longer correct for the new launcher. Settings > Guild Wars 2 > Tweaks > Launcher coordinates > configure. Ensure they're not checked.

**Comment by aZuiZui** [comment]
Hi @Healix i am using the one on the front page, the password are entered by username field is blank, so it is not continuing to the login.

**Comment by magnapeccatrix** [comment]
> Are you sure you're using the new version? The beta version would look the same as the previous, whereas the release version will have 7255 listed under Settings > Updates.
> 
> The only other reason it may do that is if the coordinates were previously configured are it's no longer correct for the new launcher. Settings > Guild Wars 2 > Tweaks > Launcher coordinates > configure. Ensure they're not checked.

Updating to this version fixed the issue for me.

**Comment by Luminariah** [comment]
> Are you sure you're using the new version? The beta version would look the same as the previous, whereas the release version will have 7255 listed under Settings > Updates.
> 
> The only other reason it may do that is if the coordinates were previously configured are it's no longer correct for the new launcher. Settings > Guild Wars 2 > Tweaks > Launcher coordinates > configure. Ensure they're not checked.

Yes, I am using the latest. I also switched back to the release version to double check to make sure if wasn't the issue - and it's still happening on release on the homepage:

![image](https://github.com/user-attachments/assets/d130da7b-78f5-4714-932a-3c41855f9b76)
